### PR TITLE
Adding Sublime Docker Image

### PR DESCRIPTION
**Pull request by harrison-cruise**
## Status
Ready

## Related Content Pull Request
No related PR

## Related Issues
No related issues

## Description
Need a docker container to integrate with sublime security products

### CODE REVIEW

File: docker/sublime/Pipfile.lock [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+            "markers": "python_version >= '3.8'",
+            "version": "==24.2.0"
+        },
+        "sublime-cli": {
+            "hashes": [
+                "sha256:652fb12b5bfaadbed9126c311026f3edfc13af035f8d2c0ec5ebd678f28c7d45",
+                "sha256:b2c80d40bae9d69321b0539e11006bdacfb5b2fc873cc8a096fa40ff12405df0"
+            ],
+            "index": "pypi",
+            "version": "==0.0.30"
```
Comment (BEAdi) [inline docker/sublime/Pipfile.lock:553]: I see that the latest version in PyPi is 0.0.32, you don't want to use it?

File: Pipfile.lock
Comment (BEAdi) [inline Pipfile.lock:1]: Restore this file

**Review by BEAdi — COMMENTED** [review]
Thank you for your contribution! We greatly appreciate it.
Please review my comments.
Could you please provide the related content PR that requires the use of this docker image? It's necessary for us in order to merge this PR.

File: docker/sublime/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+FROM demisto/python3:3.10.14.97608
```
Comment (BEAdi) [inline docker/sublime/Dockerfile:2]:
```suggestion
FROM demisto/python3:3.10.14.98471
```
We encourge to use the latest docker image.

File: docker/sublime/Pipfile.lock (hunk 1 above)
Comment (harrison-cruise) [reply docker/sublime/Pipfile.lock]: Yep, the package was updated after my initial PR.

**Review by BEAdi — APPROVED** [review]
Excellent work!
Thank you for addressing my comments.
Please share the PR containing the relevant [content](https://github.com/demisto/content) that necessitates the use of this docker image. This information is crucial for us to proceed with merging the PR.

**Comment by harrison-bornstein** [comment]
> Excellent work! Thank you for addressing my comments. Please share the PR containing the relevant [content](https://github.com/demisto/content) that necessitates the use of this docker image. This information is crucial for us to proceed with merging the PR.

Hey @BEAdi I'm not for sure what you mean? I haven't been able to build the content yet as I don't have a docker image that can run it.

**Comment by BEAdi** [comment]
Hi, you are right. I'm proceeding with merging this docker image, and we will need the content branch to be created within the following weeks. Once the branch is ready, please attach it here.

**Pull request merged**